api/jsonapi.go

```diff
@@ -2023,8 +2023,8 @@ func (i *jsonAPIHandler) POSTOpenDispute(w http.ResponseWriter, r *http.Request)
 		ErrorResponse(w, http.StatusBadRequest, "Order must be either AWAITING_FULFILLMENT or FULFILLED to start a dispute")
 		return
 	}
-	if !isSale && (state != pb.OrderState_AWAITING_FULFILLMENT && state != pb.OrderState_PENDING && state != pb.OrderState_FULFILLED) {
-		ErrorResponse(w, http.StatusBadRequest, "Order must be either AWAITING_FULFILLMENT, PENDING, or FULFILLED to start a dispute")
+	if !isSale && !(state == pb.OrderState_AWAITING_FULFILLMENT || state == pb.OrderState_PENDING || state == pb.OrderState_FULFILLED || state == pb.OrderState_PROCESSING_ERROR) {
```

Comment:
If the state is PROCESSING_ERROR, shouldn't we also ensure the order is at least partially funded? In other words, do we want to let them open a dispute if the order is not funded at all?